cirq-core/cirq/protocols/channel_test.py

```diff
@@ -52,9 +52,9 @@ def assert_not_implemented(val):
     assert not cirq.has_channel(val)
 
 
-def test_channel_returns_not_implemented():
+def test_kraus_returns_not_implemented():
```

Comment:
We'll also need backwards-compatibility tests for classes using the `_channel_` method.

Reply:
Added tests for fallback and deprecation warnings.